neuron_morphology/pipeline/post_data_to_s3.py

```diff
@@ -0,0 +1,145 @@
+import os
+import boto3
+import logging
+import json
+from botocore.exceptions import ClientError
+from zipfile import ZipFile, ZIP_DEFLATED
+from io import BytesIO
+from argschema import ArgSchemaParser
+from neuron_morphology.pipeline._schemas import InputParameters
+
+def post_file_to_s3(file_name, bucket_name):
+    """
+    This is to post data to S3 bucket
+    
+    Parameters
+    ------------------
+    file_name: the file need to be uploaded
+    bucket_name: s3 bucket name
+    
+    Return
+    ------------------
+    True if successful
+
+    """
+    aws_id = os.getenv('aws_access_key_id')
```

Comment:
neuron_morphology requires latest allensdk. Any reason you are not using the allensdk credential manager that @kschelonka wrote?

Reply:
Can you show me an example how to use allensdk credential?

Reply:
changed. boto3 don't need explicitly set those credential parameters.